ethcore/src/client/client.rs

```diff
@@ -648,7 +648,7 @@ impl BlockChainClient for Client {
 			timestamp: view.timestamp(),
 			difficulty: view.difficulty(),
 			last_hashes: last_hashes,
-			gas_used: U256::zero(),
+			gas_used: Default::default(),
```

Comment:
TBH I prefer more explicit approach in such cases (`U256::default(), U256::from(0)`), for me it just means less cognitive load.

Look at `transaction_hash` below especialy, It's quite difficult to understand what type is returned (it gets even more tricky when there is some more `.map`ing on options/results/vectors done).

But it's just an opinion :) I'm good with the changes :+1: